### ISSUE TITLE
GBIF file questions

### ISSUE THREAD

**Issue by seltmann** [seltmann/allArthropodDataProject#6]
@CatChapman @neilcobb 
Regarding the GBIF zip files. Are these already cleaned up? With redundant data removed? If they are not the cleaned files can you send me your search and filter criteria for looking for redundant data?

Also, are we including observations in this dataset? I assume that we are only calculating for records with the dwc:basisofrecord=preserved specimens?

**Comment by seltmann** [comment]
I downloaded all the files already so feel free to take them off of dropbox

**Comment by neilcobb** [comment]
They are not cleaned and I am not sure if Lindsie and Cat finished making the list of institution codes.

**Comment by seltmann** [comment]
Are we calculating only on specimens or are we going to include observations? I think we only used specimens last time...

**Comment by neilcobb** [comment]
Good point and that might be the difference, I forgot to screen for observations.  We are just doing specmens but now I really want to see the observations.

**Comment by CatChapman** [comment]
Good evening Katja!

I just sent Neil a list of institutions in SCAN + their GBIF codes. Not many of our institutions have published datasets on SCAN so my work was easier than I initially thought. Hopefully what I've done is workable and makes it to your neck of the woods soon. :) 

I have been using this tool to find data publishers on GBIF http://www.gbif.org/publisher/search but there are definitely codes that I have seen - NMSU for example - that are not listed on the provided link. Legacy data??? I'm not sure. So be wary of that... but I think it's thorough enough, based on the literal millions of records I have since manually gone thru in cleaning up a handful of GBIF datasets.

**Comment by neilcobb** [comment]
Yep, it is hard to find some of these legacy codes, but I assume that they will not matter that much

**Comment by seltmann** [comment]
Hi Cat!
Thanks for all your help and I look forward to that list. I saw that you are on for the idigbio API hackathon. Are you involved in a project yet? I know Francois has been working on compiling a master institution list for idigbio but i dont know about the progress. Something like that would so help us all out and seems like a good hackathon topic! Unfortunately, I wont be there so we wont meet in person.

**Comment by seltmann** [comment]
Neil at least we can find out how many observations vs specimens and other basis of record are in gbif at the moment. Likely then I will pull them out and continue the analysis using only specimen data for now.

The prize for insect observations maybe bugguide. Although there is no easy way to get to those records and will require some fancy data mining. Great student project perhaps?

**Comment by neilcobb** [comment]
Tried calling



Sent from my Verizon Wireless 4G LTE smartphone


-------- Original message --------

**Comment by neilcobb** [comment]
I just realized that I have no idea who is linked on these emails.  However, Ben and Ed will be there and the primary goal for me is to have her attack an iDigBio API that she can work on that will increase functionality for Symbiota.  We will talk toady and I hope that she and Ben can Skype next week so her experience at the workshop is maximized.

**Comment by seltmann** [comment]
@CatChapman
One further question. When you upload data to SCAN from GBIF do you use the occurrence.txt file or the verbatim.txt file. I am planning to use the occurrence.txt file because it has already been run through GBIF cleaning services.
